src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -0,0 +1,221 @@
+            }
+
+            VPNVerticalSpacer {
+                Layout.fillWidth: true
+                Layout.minimumHeight: VPNTheme.theme.rowHeight
+            }
+        }
+    }
+
+    function handleManageAccountClicked() {
+        Sample.manageAccountClicked.record();
+        VPN.openLink(VPN.LinkAccount);
+    }
+
+    function populateListModels() {
+        // Subscription info model
+        subscriptionInfoModel.append({
+            labelText: VPNl18n.SubscriptionManagementPlanLabel,
+            valueText: getPlanText(
+                VPNSubscriptionData.planCurrency,
+                VPNSubscriptionData.planAmount,
+                VPNSubscriptionData.planIntervalCount
+            ),
+            type: "text",
+        });
+
+        subscriptionInfoModel.append({
+            labelText: VPNl18n.SubscriptionManagementStatusLabel,
+            valueText: VPNSubscriptionData.status,
+            type: "pill",
+        });
+
+        subscriptionInfoModel.append({
+            labelText: VPNl18n.SubscriptionManagementNextLabel,
+            // TODO: Calculate next billing date if any
+            valueText: "",
+            type: "text",
+        });
+
+        subscriptionInfoModel.append({
+            labelText: VPNl18n.SubscriptionManagementActivatedLabel,
+            valueText: unixToDate(VPNSubscriptionData.createdAt),
+            type: "text",
+        });
+
+        subscriptionInfoModel.append({
+            labelText: VPNl18n.SubscriptionManagementExpiresLabel,
+            valueText: unixToDate(VPNSubscriptionData.expiresOn),
+            type: "text",
+        });
+
+        // Subscription payment model
+        if (VPNSubscriptionData.paymentType === "credit") {
+            subscriptionPaymentModel.append({
+                labelText: VPNSubscriptionData.creditCardBrand,
+                valueText: VPNl18n.SubscriptionManagementCardLast4.arg(VPNSubscriptionData.creditCardLast4),
+                type: "creditCard",
+            });
+
+            subscriptionPaymentModel.append({
+                labelText: VPNl18n.SubscriptionManagementCardExpiresLabel,
+                // TODO: Show date w/o days
+                valueText: unixToDate(new Date(VPNSubscriptionData.creditCardExpYear, VPNSubscriptionData.creditCardExpMonth - 1).getTime() / 1000),
+                type: "text",
+            });
+        } else {
+            subscriptionPaymentModel.append({
+                // TODO: Handle different IAP platforms
+                labelText: VPNl18n.SubscriptionManagementSubscriptionPlatformLabel.arg("IAP Google or Apple"),
+                valueText: "",
+                type: "text",
+            });
+        }
+    }
+
+    function unixToDate(unixTimestamp) {
+        return new Date(unixTimestamp * 1000).toLocaleDateString(currentLocale, Locale.ShortFormat);
+    }
+
+    function getPlanText(currencyCode, amount, intervalCount) {
+        const amountDisplay = (amount || 0) / 100;
```

Comment:
why `/100` ?

Reply:
We always get the amount in the smallest currency unit (for USD it’s cents). FxA does the same for their display amount here in their [getLocalizedCurrencyString](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/src/lib/formats.ts#L40-L48) as well.